bittensor/_neuron/text/core_server/nucleus_impl.py

```diff
@@ -78,6 +84,9 @@ def __init__(self,
         if self.config.neuron.autocast and self.device[:4] == 'cuda':
             self.pre_model.half()
 
+        # Generative default expects most recent token on right-hand side with padding on left. https://github.com/huggingface/transformers/pull/10552
+        self.tokenizer.padding_side = "left"
```

Comment:
Can we use the bittensor.tokenizer.prep_tokenizer function? Just for consistency and in case we write more miners in the future.

Reply:
Yes, we can use prep_tokenizer, will do it in another pull request.